### PR TITLE
SMC feature HCM- 611,865,868

### PR DESCRIPTION
**Pull request by amitverma-egov**
1)Replace the label "ineligible" for "householder head" - hcm-611
2)Add a field to capture if there were a wasted SPAQ - hcm -865
3)Update the labels in inventory flow to match the NCMP materials and forms - hcm-868

### CODE REVIEW

File: apps/health_campaign_field_worker_app/lib/pages/beneficiary/deliver_intervention.dart [hunk 1]
```diff
@@ -225,7 +242,33 @@ class _DeliverInterventionPageState
...
+                                                    context,
+                                                    options: DigitToastOptions(
+                                                      localizations.translate(i18
+                                                          .deliverIntervention
+                                                          .deliveryCommentRequired),
+                                                      true,
+                                                      theme,
+                                                    ),
+                                                  );
+                                                }else {
```
Comment (vishal-egov) [inline apps/health_campaign_field_worker_app/lib/pages/beneficiary/deliver_intervention.dart:271]: This validation is not required for this field @amitverma-egov

File: apps/health_campaign_field_worker_app/lib/pages/beneficiary/deliver_intervention.dart [hunk 2]
```diff
@@ -199,6 +200,22 @@ class _DeliverInterventionPageState
...
+                                                  await DigitToast.show(
+                                                    context,
+                                                    options: DigitToastOptions(
+                                                      localizations.translate(i18
+                                                          .deliverIntervention
+                                                          .resourceCannotBeZero),
+                                                      true,
+                                                      theme,
+                                                    ),
+                                                  );
```
Comment (vishal-egov) [inline apps/health_campaign_field_worker_app/lib/pages/beneficiary/deliver_intervention.dart:218]: We need to confirm once if the field is mandatory @amitverma-egov

File: apps/health_campaign_field_worker_app/lib/pages/beneficiary/deliver_intervention.dart [hunk 3]
```diff
@@ -548,6 +597,8 @@ class _DeliverInterventionPageState
         .add(FormControl<ProductVariantModel>());
     (form.control(_quantityDistributedKey) as FormArray)
         .add(FormControl<String>(validators: [Validators.required]));
+    (form.control(_quantityWastedKey) as FormArray)
+        .add(FormControl<String>(validators: [Validators.required]));
```
Comment (vishal-egov) [inline apps/health_campaign_field_worker_app/lib/pages/beneficiary/deliver_intervention.dart:601]: We need to confirm if field is mandatory @amitverma-egov

File: apps/health_campaign_field_worker_app/lib/pages/beneficiary/deliver_intervention.dart [hunk 4]
```diff
@@ -642,6 +693,11 @@ class _DeliverInterventionPageState
             AdditionalFieldsType.deliveryStrategy.toValue(),
             deliveryStrategy,
           ),
+          AdditionalField(
+            _quantityWastedKey,
+            form.control(_quantityWastedKey)
+                    .toString(),
+          ),
```
Comment (vishal-egov) [inline apps/health_campaign_field_worker_app/lib/pages/beneficiary/deliver_intervention.dart:700]: If the field is not mandatory add only if it is not null @amitverma-egov

File: apps/health_campaign_field_worker_app/lib/widgets/member_card/member_card.dart [hunk 5]
```diff
@@ -175,7 +175,7 @@ class MemberCard extends StatelessWidget {
                         icon: Icons.info_rounded,
                         iconSize: 20,
                         iconText: localizations.translate(
-                          (isNotEligible || isBeneficiaryIneligible)
+                          isHead ? i18.householdOverView.householdOverViewHouseholderHeadLabel : (isNotEligible || isBeneficiaryIneligible)
```
Comment (vishal-egov) [inline apps/health_campaign_field_worker_app/lib/widgets/member_card/member_card.dart:178]: We also need to hide the action buttons when it is a head, lets confirm this as well @amitverma-egov

**Review by vishal-egov — CHANGES_REQUESTED** [review]

File: apps/health_campaign_field_worker_app/lib/widgets/member_card/member_card.dart (hunk 5 above)
Comment (amitverma-egov) [reply apps/health_campaign_field_worker_app/lib/widgets/member_card/member_card.dart:178]: done

**Review by vishal-egov — APPROVED** [review]

**Pull request merged**